sdk/core/azure-core-xml/vcpkg/vcpkg.json

```
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+{
+  "$comment": [
+    "NOTE: All changes made to this file will get overwritten by the next port release.",
+    "Please contribute your changes to https://github.com/Azure/azure-sdk-for-cpp."
+  ],
+  "name": "azure-core-xml-cpp",
+  "version-semver": "@AZ_LIBRARY_VERSION@",
+  "description": [
+    "Azure Core Xml provides an implementation of serializing/deserializing Xml documents in C++."
+  ],
+  "homepage": "https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/",
+  "license": "MIT",
+  "dependencies": [
+    {
+      "name": "azure-core-cpp",
+      "default-features": false,
+      "version>=": "1.3.1"
```

If you haven't tested it will work with 1.3.1 (a version released in November 2021), better put the latest GA version.
```suggestion
      "version>=": "1.10.3"
```